docs/crud.md

```diff
@@ -0,0 +1,103 @@
+# Access a document
+
+#### Request
+```http
+GET /data/:type/:id
+```
+```http
+GET /data/types.cozy.io/events/6494e0ac-dfcb-11e5-88c1-472e84a9cbee
```

Comment:
The all types should be fully qualified to distinguish between types.cozy.io/messages (email) and my-great-irc-app.xyz/messages for instance.